docs/00_introduction/motivation.md

```diff
@@ -0,0 +1,17 @@
+# Motivation Behind Garden Linux
+
+Garden Linux was conceived with a vision to address the evolving needs of modern infrastructure. It's not just about creating another Linux distribution; it's about crafting an OS that's adaptable, efficient, and secure. 
+
+## Core Concepts
+
+### **Efficient Build System**
+- **Builder (Meta Package Manager)**: Instead of reinventing the wheel, Garden Linux continues to work with existing package managers. What's innovative is how we use them, leading to the creation of the builder. It's about reimagining the usage, not the tool itself.
```

Comment:
this whole paragraph is a bit too fuzzy for my taste

I think it is good to highlight how the builder is super generic and allows to build custom linux systems with pluggable features, but I don't understand the 'reinventing the wheel' part and I'm not sure if I get what exactly the innovation is (compared to other approaches of linux builders)

Reply:
Agree, will do this in a follow up PR.